xds/internal/client/xds.go

```diff
@@ -634,15 +658,17 @@ func validateCluster(cluster *v3clusterpb.Cluster) (ClusterUpdate, error) {
 	if env.ClientSideSecuritySupport {
 		var err error
 		if sc, err = securityConfigFromCluster(cluster); err != nil {
-			return emptyUpdate, err
+			return ClusterUpdate{ServiceName: "", EnableLRS: false}, err
```

Comment:
This change is unnecessary, right?

Reply:
Ugh, I forgot to change it back. Changed back.